src/components/RockPaperScissors/SelectState.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react'
+import { BottomSelect, MiddleSelect, Overlay, SelectContainer, TopSelect } from './styles/SelectState.styled'
+import pentagonIcon from '../../assets/RockPaperScissors/bg-pentagon.svg'
+import { Option } from '../../pages/RockPaperScissors/util'
+
+type Props = {
+  renderButton: (option: Option) => ReactElement
+  optionList: Option[]
+}
+
+const SelectState = ({ renderButton, optionList }: Props) => (
```

Comment:
You might want to change name of the props renderButton as it is not an actual button.